gems/bundled_gems

```diff
@@ -1,7 +1,8 @@
+# gem-name version-to-bundle repository-url [optional-commit-hash-to-test-or-defaults-to-v-version]
 minitest 5.14.2 https://github.com/seattlerb/minitest
 power_assert 1.2.0 https://github.com/ruby/power_assert
 rake 13.0.1 https://github.com/ruby/rake
-test-unit 3.3.6 https://github.com/test-unit/test-unit
+test-unit 3.3.6 https://github.com/test-unit/test-unit 3.3.6
```

Comment:
We need to specify the tag to test `test-unit` gem explicitly because the repository doesn't have `v3.3.6` tag. 
The `fetch-bundled_gems` had tried checking out `3.3.6` tag too, but it's removed.